### ISSUE TITLE
Just wanted to say awesome work

### ISSUE THREAD

**Issue by binaryfire** [repman-io/repman#112]
Hey guys

Ok this isn't an issue, but just wanted to say this is a game changed for PHP devs. I've spread the word in the WordPress community so hopefully you'll see a bunch of new users soon. 

Keep up the great work and thanks for making this open source.

**Comment by akondas** [comment]
Thank you :clap: very much for your kind words, if you have any questions or ideas, please go ahead and open a new issue :wink:

**Comment by akondas** [comment]
You can always follow us on Twitter: https://twitter.com/repman_io

**Comment by Radiergummi** [comment]
Yeah @akondas I'd also like to say thank you so much for this! I've dealt with Satis too long, and now Repman is here :tada:

**Comment by akondas** [comment]
@Radiergummi thanks, but I'm not alone, @karniv00l also works directly with me, and of course rest of our awesome team. On behalf of everyone, thanks :wink:

**Comment by xleliberty** [comment]
Yep, i want to associate to this message.
Thank you very much for this fresh alternative to satis !
